pyproject.toml

```diff
@@ -71,20 +71,25 @@ testpaths = "assesspy/tests"
 legacy_tox_ini = """
 [tox]
 min_version = 4.0
-envlist = py39, py38, py310, py311, py312
+envlist = py39, py310, py311, py312
 
-[gh]
-python =
-    3.12 = py312
-    3.11 = py311
-    3.10 = py310
-    3.9 = py39
-    3.8 = py38
```

Comment:
We no longer need this environment mapping since we just explicitly specify the tox environment via these commands:

```bash
env=$(echo ${{ matrix.python-version }} | tr -d '.' | sed 's/^/py/')
tox r -v --notest -e "$env"
```